database/frontend/design-inspirations.json

```diff
@@ -1,15 +1,101 @@
-[{
-        "name": "Dribbble",
-        "description": "Explore the world’s leading design portfolios.",
-        "url": "https://dribbble.com",
-        "category": "frontend",
-        "subcategory": "design-inspiration"
-    },
-    {
-        "name": "Godly Website",
-        "description": "A curated collection of good web design from all over the web",
-        "url": "https://godly.website",
-        "category": "frontend",
-        "subcategory": "design-inspiration"
-    }
-]
\ No newline at end of file
+[
+  {
+    "name": "Dribbble",
+    "description": "Explore the world’s leading design portfolios.",
+    "url": "https://dribbble.com",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Godly Website",
+    "description": "A curated collection of good web design from all over the web",
+    "url": "https://godly.website",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Behance",
+    "description": "Behance is the world's largest creative network for showcasing and discovering creative work.",
+    "url": "https://www.behance.net/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Refero Design",
+    "description": "Get inspired by the best examples of product design.",
+    "url": "https://refero.design/",
+    "category": "frontend",
+    "subcategory": "design-inspiration"
+  },
+  {
+    "name": "Page Collective",
+    "description": "Get inspired and keep up with the latest landing page design trends.",
```

Comment:
Great start @Oyebolade! However, I am curious about what makes Page Collective distinctive and more beneficial than the other sites you mentioned. Consider adding this to your description.